runtime/oti/cfr.h

```diff
@@ -865,7 +865,11 @@ typedef struct J9CfrClassFile {
 #define CFR_MAJOR_VERSION  45
 #define CFR_MINOR_VERSION  3
 #define CFR_PUBLIC_PRIVATE_PROTECTED_MASK	(CFR_ACC_PUBLIC | CFR_ACC_PRIVATE | CFR_ACC_PROTECTED)
-#define CFR_CLASS_ACCESS_MASK					(CFR_ACC_PUBLIC | CFR_ACC_FINAL | CFR_ACC_SUPER | CFR_ACC_INTERFACE | CFR_ACC_ABSTRACT | CFR_ACC_SYNTHETIC | CFR_ACC_ANNOTATION | CFR_ACC_ENUM | CFR_ACC_MODULE)
+#if JAVA_SPEC_VERSION >= 9
```

Comment:
We're not using compile time checks yet.  Please do this with a runtime check at the relevant callsites.